include/n64dd.h

```diff
@@ -1,10 +1,15 @@
 #ifndef N64DD_H
 #define N64DD_H
 
+#include "z64play.h"
+#include "z64scene.h"
+
 typedef struct n64ddStruct_80121AF0 {
     char unk_00[0x3C];
     void (*unk_3C)(void);
     void (*unk_40)(void);
+    char unk_44[0x28];
+    void (*unk_6C)(PlayState*, SceneDrawConfigFunc*);
```

Comment:
`SceneDrawConfigFunc` is a pointer type already, so there shouldn't be a need for the `*`, right?
```suggestion
    void (*unk_6C)(PlayState*, SceneDrawConfigFunc);
```

Reply:
this function is actually passed the whole array of scene draw function pointers

Reply:
Oh, whoops, I completely missed it, sorry